drivers/boston_project_scorecard/app/mailers/boston_project_scorecard/scorecard_mailer.rb

```diff
@@ -12,9 +12,9 @@ def scorecard_prefilled(report, contact)
     mail(to: @contact.email, subject: "Scorecard For #{@report.project_name}")
   end
 
-  def scorecard_ready(report, contact)
+  def scorecard_ready(report)
     @report = report
-    @contact = contact
+    @contact = @report.secondary_contact
```

Comment:
pretty sure you meant:
```suggestion
    @contact = @secondary_reviewer
```